packages/ui/src/components/va-data-table/VaDataTable.vue

```diff
@@ -533,25 +533,25 @@ export default defineComponent({
       }
     }
 
-    .va-data-table__table-tr {
-      position: relative;
-      z-index: 0;
-
-      &.selected {
-        background-color: var(--va-selected-color);
-      }
```

Comment:
Table rows can be `selected` not only when table is `striped`. Please don't remove this from here.